packages/oncokb-ts-api-client/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "oncokb-ts-api-client",
+  "description": "OncoKB API Client for TypeScript",
+  "version": "0.1.0",
```

Comment:
This version should reflect the actual API version, ideally the latest tag version on oncokb master branch

Reply:
after discussing with @zhx828, decided to set the version to the current version in [swagger](https://www.oncokb.org/swagger-ui/index.html) doc, which is `1.0.0`